src/api/api.ts

```diff
@@ -1,42 +1,7 @@
-const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Reusability: The BASE_URL is used in multiple functions, so it could be defined in a single place and imported where needed.